packages/cli/src/commands/versions/bump.ts

```diff
@@ -141,9 +164,42 @@ export default async () => {
     });
 
     console.log();
-    console.log("Running 'yarn install' to install new versions");
+    console.log(
+      `Running ${chalk.blue('yarn install')} to install new versions`,
+    );
     console.log();
     await run('yarn', ['install']);
+
+    if (breakingUpdates.size > 0) {
+      console.log();
+      console.log(
+        chalk.yellow('The following packages may have breaking changes:'),
```

Comment:
The word "breaking" to me really should make it stand out more. For accessibility, I can appreciate red text on black background can be hard, but almost wonder if this should include a red X emoji or something to make it stand out for the user.